lib/octokit/client/repositories.rb

```diff
@@ -155,6 +156,7 @@ def create_repository(name, options = {})
         opts = options.dup
         organization = opts.delete :organization
         opts.merge! :name => name
+        ensure_api_media_type(:template_repositories, opts) if opts.include? :is_template
```

Comment:
This is where I'm at a crossroads.
Adding support for the preview in our existing method `create_repository` vs. isolating it into its own method (e.g. `create_template_repository`).
You can see I did this for `template_repository` - I could've easily implemented it in the main `edit_repository` method like I did here with `create_repository`.
What do you think @tarebyte?

Reply:
I'm 👍 for updating the existing methods `create_repository` and `edit_repository` rather than creating a new method just for this option.

It keeps things more consistent and I think it would be an unexpected side effect if these methods didn't support these options.